### DISCUSSION TITLE
Is it possible to compile for ARM64/AARCH64 linux?

### DISCUSSION THREAD

**Discussion by dylan-hoefsloot** [wezterm/wezterm#967] [Q&A] [answered]
I can use it on my ARM Macbook, however, I am having trouble running on Alpine Linux. I have tried the AppImage, however that doesn't work. I have also tried downloading the source and running `cargo build --release`, however this generates error and I am not familiar with rust, Ideally I would like to download some sort of precompiled executable like the AppImage.

**Comment by wez** [comment, marked as answer]
My understanding is that wezterm builds on aarch64 linux, but I don't use it myself.  I'm not opposed to someone contributing github CI changes to add an aarch64 build, but don't have the time for it myself.

If you share the error message that you saw, I might be able to help you figure out how to resolve your build issue.
